### PR TITLE
feat: landing page, new assets, and favicon update

### PR DESCRIPTION
**Pull request by d-x-s**
https://github.com/d-x-s/airon-fitness/assets/68035358/3bee1335-294b-4baf-a22f-0689e1504fb7

image

- Improve button styling and overall page responsiveness

- Experiment with landing page to find a good palette (I know we voted on one earlier but I'm finding gold easier to work with then the pink/purple shades as the accent color).

- Upload all experimental assets

- Update favicon to use proposed logo

### CODE REVIEW

**Comment by jakji93** [comment]
I like the gold color here

File: server/src/utils/prompts/mealPrompts.js [hunk 1]
```diff
@@ -1,52 +1,72 @@
...
-
-            Please make the necessary modifications and return the updated workout schedule in the same JSON format. Thank you!
-            Don't include Note. For null values use 0 instead.`  
-  }
-
-  module.exports = {
-    mealPrompt,
-  }
\ No newline at end of file
+const mealCreationPrompt = (user) => `
```
Comment (jakji93) [inline server/src/utils/prompts/mealPrompts.js:1]: I see the updated prompt is also fixed into this pr, I'll approve this one as well

**Review by jakji93 — APPROVED** [review]

File: server/src/utils/prompts/mealPrompts.js (hunk 1 above)
Comment (d-x-s) [reply server/src/utils/prompts/mealPrompts.js:1]: Ohhhh I must have branched off the branch. If we merge in order things should be OK

File: client/src/pages/Landing.jsx [hunk 2]
```diff
@@ -81,9 +105,14 @@ export default function Landing() {
...
           </Typography>
         </div>
-        <Button variant="outlined" fontWeight="light" style={styles.actionButton} href="/login">
-          Let&apos;s get started.
-        </Button>
+        <div className="button-container" style={styles.buttonContainer}>
+          <Button variant="outlined" sx={styles.actionButton} href="/login">
+            Let&apos;s get started.
+          </Button>
+          <Button variant="outlined" sx={styles.actionButton} href="/app/about">
```
Comment (kennyjhcheng) [inline client/src/pages/Landing.jsx:112]: Can you move the about page out of the `/app/` route, so href would be `/about`.

Everything under `/app` should be blocks by logging in, but I think about should be visible without logging in

**Review by kennyjhcheng — CHANGES_REQUESTED** [review]

File: client/src/pages/Landing.jsx (hunk 2 above)
Comment (kennyjhcheng) [reply client/src/pages/Landing.jsx:112]: ill approve right after this change is made, just ping me on slack

**Review by jakji93 — APPROVED** [review]

**Review by kennyjhcheng — APPROVED** [review]

**Pull request merged**